os/arch/arm/src/amebalite/amebalite_rtc.c

```diff
@@ -118,7 +118,7 @@ int up_rtc_gettime(FAR struct timespec *tp)
 {
 	time_t timer;
 	int ret;
-	timer = rtk_rtc_read();
+	timer = rtc_read();
```

Comment:
Is this necessary? Could you let me know why this is changed?

Reply:
Because in amebalite code the API's are rtc_xxx and not rtk_rtc_xxx
https://github.com/Samsung/TizenRT/blob/cd73fabd417d6d43d5b19bbd84c6254dd072e0e4/os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/rtc_api.c#L255

Reply:
If I leave it as such, it will generate build breaks with RTC enabled.